scripts/changelog-links.sh

```diff
@@ -1,31 +0,0 @@
-#!/bin/bash
```

Comment:
Removing this seems fine to me. I don't believe we ever used it. 

GitHub automatically does what this intended too: https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/autolinked-references-and-urls